README.md

```diff
@@ -1,2 +1,162 @@
-# asgardeo-nextjs-b2b-sample-app
-Next.JS sample web application to integrate with Asgardeo for B2B usecases.
+## 👀 Live Deployment
+
+A live preview of this demo is available at
+
+1. Vercel: [https://asgardeo-b2b-nextjs-sample-app.vercel.app/](https://asgardeo-b2b-nextjs-sample-app.vercel.app/)
+
+Sample Credential
+
+- organization name : Best Car Mart
+- username : Alex@gmail.com
+- password : Alex@123
+
+## Features/ Capabilities
+
+- Sign In to the sub organization application
+- Sign Out of the application
+- User profile view and update
+- User Management.
+  - View all users
+  - Add new user
+  - Edit user
+  - Delete User
+- Group Management
+  - View groups
+  - Add a new group and assign users.
+  - Edit group and edit assigned users.
+  - Delete group
+- Role Management
+  - View Role permission
+  - View assigned groups
+  - Assign new groups
+  - Assign external groups
+- Idp Management
+  - Add Idp ( Google Idp/ Enterprise Idp)
+  - Edit Idp settings
+  - Add external groups
+  - Add Idp to application login flow
+
+## Configure Asgardeo to integrate with your application
+
+Create a sub organization
```

Comment:
Shall we use the h3 notation for sub titles?

```suggestion
### Create a sub organization
```

Reply:
Fixed